syntax = "proto3";

package tasknotes.common;

option go_package = "github.com/tasknotes/proto/common";

// Tipos comunes compartidos entre servicios

message CategorySummary {
  string id = 1;
  string name = 2;
  string color = 3;
}

message TagSummary {
  string id = 1;
  string name = 2;
  string color = 3;
}